scripts/pyproject.toml

```diff
@@ -26,5 +26,11 @@ select = [
   "RUF",
 ]
 
+ignore = [
+  # only relevant if you run a script with `python -0`,
+  # which seems unlikely for any of the scripts in this directory
+  "B011"
+]
```

Comment:
This is no longer necessary now I'm using `assert_never` in the script rather than a raw `assert False`, but I think it makes sense to ignore this rule in the `scripts/` directory regardless